redux/slices/admin.ts

```diff
@@ -0,0 +1,24 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+
+export type Admin = "true" | "false";
+
+export const defaultPermissions = "false";
```

Comment:
😢

Reply:
I will fix and remake to boolean, sure it will be better